README.rst

```diff
@@ -30,6 +30,10 @@ FAQ
 
   A- The `industrial_ci` still provides valuable checks; it ensures if your package builds without issues. Also installation rules if you define. Just as a headsup that making test cases are highly recommended as your ear may hurt.
 
+- Q- My package uses a custom Point Cloud Library (PCL) version or the [`industrial_calibration`](https://github.com/ros-industrial/industrial_calibration) package, how do I make build work?
```

Comment:
This is `rst` file so link needs to be something like:

    `industrial_calibration <https://github.com/ros-industrial/industrial_calibration>`_